### PR TITLE
es6-operators - Roland Estropia Jr.

### PR DESCRIPTION
**Pull request by restropiajr**
image

### CODE REVIEW

File: es6-operators/main.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+ * `oc1`, but only if it is defined.
+ * Do the same for the variable `oc2` and the variable v2.
+ * Do the same for the variable `oc3` and the variable v3.
+ * Log `oc1`, `oc2`, and `oc3`.
+ * Expected: `oc1 = 'Something'`, `oc2 = undefined`, `oc3 = undefined`.
+ */
+
+const oc1 = v1?.value ?? undefined;
+const oc2 = v2?.value ?? undefined;
+const oc3 = v3?.value ?? undefined;
```
Comment (shawnkost) [inline es6-operators/main.js]: No `??` here. Just `?.`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Roland!

File: es6-operators/main.js (hunk 1 above)
Comment (restropiajr) [reply es6-operators/main.js]: image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**